modules/uhttp/handler.go

```diff
@@ -58,7 +59,7 @@ type handlerWrapper struct {
 // ServeHTTP calls Handler.ServeHTTP(ctx, w, r) and injects a new service context for use.
 func (h *handlerWrapper) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx := fx.NewContext(context.Background(), h.host)
-	stopwatch := h.host.Metrics().Timer("http." + r.Method + ".time").Start()
+	stopwatch := stats.HTTPMethodTimer[r.Method].Start()
```

Comment:
I am really curious, how thread safe is it..
Theoretically it should be fine, but I'd prefer to test it.